libs/common/src/constants/portal/cover-end-date.ts

```diff
@@ -0,0 +1 @@
+export const COVER_END_DATE_MAXIMUM_YEARS_IN_FUTURE = 6 as const;
```

Comment:
Can we add a comment regarding 6 being a business requirement rather than an arbitary number please.